scripts/prepare-machine.ps1

```diff
@@ -251,7 +246,7 @@ function Install-Xdp-Driver {
     }
 
     Write-Host "Installing XDP driver"
-    netcfg.exe -l "$XdpPath\bin\xdp.inf" -c s -i ms_xdp
+    msiexec.exe /i $XdpPath\bin\xdp-for-windows.msi /quiet
```

Comment:
So this automatically uninstalls anything previously installed? Is that why you removed the delete above?

Reply:
It uninstalls anything that has been installed using this package. It would be an error for something else to leave XDP installed.